pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/answer/DecimalAnswer.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.ddp.model.activity.instance.answer;
+
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.model.activity.definition.types.DecimalDef;
+import org.broadinstitute.ddp.model.activity.types.QuestionType;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+public final class DecimalAnswer extends Answer<DecimalDef> {
+    @SerializedName("value")
+    private DecimalDef value;
+
+    public DecimalAnswer(Long answerId, String questionStableId, String answerGuid, DecimalDef value) {
+        super(QuestionType.DECIMAL, answerId, questionStableId, answerGuid);
+        this.value = value;
+    }
+
+    public DecimalAnswer(Long answerId, String questionStableId, String answerGuid, DecimalDef value, String actInstanceGuid) {
+        super(QuestionType.DECIMAL, answerId, questionStableId, answerGuid, actInstanceGuid);
+        this.value = value;
+    }
+
+    @Override
+    public DecimalDef getValue() {
+        return value;
+    }
+
+    @Override
+    public void setValue(DecimalDef value) {
+        this.value = value;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return value == null;
+    }
```

Comment:
The answer definition differs from what we have in ReDoc:
![Screen Shot 2022-02-07 at 12 24 57 PM](https://user-images.githubusercontent.com/29984380/152840038-3092ee4c-e803-4373-b047-931c631c47d5.png)
@bskinner @alextanlan  Does it make sense to have those additional attributes (precision and scale) in the answer? (i.e., wouldn't the client just send back whatever the user entered and just echo back the precision and scale values to make the API happy).
The specific question: can we drop precision and scale from the answer attributes in the API?

Reply:
Precision is purely for display & validation purposes and only needs to be present in the question  definition- it's the server's responsibility to validate that and, as such, isn't necessary in the answer (and isn't present in the current docs & implementation). As for the scale- that's an intrinsic part of the BigDecimal type, and the number can not be represented without it. We could send the representation as a string, but that assumes whatever client libraries we are interacting with will be able to encode and decode a strings representation of a BigDecimal, and that's not a given.

Reply:
Oh wait: the type for the answer is int64? Is just using the "number" type a problem?

Reply:
Yes, it is. RFC-8259 does not define a representation of the `number` type but, in practice, most systems use the interpretation of a `number` type as either an integer, or a IEEE 754 double-precision 64-bit floating point value (including the ECMAScript standard)

GSON is a bit of an outlier here in that it provides an API for directly converting a JSON Number to a BigDecimal value, which most other libraries do not provide.

Reply:
Oh, huh- I just took another look at your screenshot. The `precision` property there _is_ an error, and I'll get that fixed up ASAP.

Reply:
OK. So thinking how this will work in Angular:
User enters a number (e.g, "4.75") in the input field.
Client would then: count the digit characters after the . (in this case, 2).
Transform to a string without the . character => "475"
send answer: 
```json
{answerGuid: "blah", value: 475, scale: 2}
```
Am I getting this right?

Reply:
Yup- that's the general idea.

Reply:
Yep. The example is in Basil study in this PR...